AUTHORS

```diff
@@ -0,0 +1,3 @@
+Adir Gabai <adir@stratoscale.com>
+Eran <eran@stratoscale.com>
```

Comment:
The file was generated automatically. I can change it manually, but i guess it will be overridden. 
@eran-stratoscale did you git-configure your surname?